src/ray/gcs/tables.cc

```diff
@@ -3,6 +3,11 @@
 #include "common_protocol.h"
 #include "ray/gcs/client.h"
 
+namespace {
+static const std::string kTableAppendCommand = "RAY.TABLE_APPEND";
```

Comment:
Should we also move the `TABLE_ADD` commands here?

Reply:
Not really since that code is not duplicated.  Also I don't see a good way to abstract that piece across append and add, without doing costly / unwieldy string concatenation.  Here the strings are compile-time constants.

Reply:
Hmm, I just meant to also define `static const std::string`s for the `TABLE_ADD` commands.

I'm fine with not trying to abstract it out right now, but in the future we could still define the strings statically while doing the concatenation in the definition. As we add more commands, it would be nice to have a static table of commands that we could lookup as part of the helper function that you added in this PR.

Reply:
Done